includes/Article.php

```diff
@@ -1278,64 +1278,65 @@ public function setOldSubtitle( $oldid = 0 ) {
 	}
 
 	/**
-	 * View redirect
+	 * Return the HTML for the top of a redirect page
 	 *
-	 * @param $target Title|Array of destination(s) to redirect
-	 * @param $appendSubtitle Boolean [optional]
-	 * @param $forceKnown Boolean: should the image be shown as a bluelink regardless of existence?
-	 * @return string containing HMTL with redirect link
+	 * Chances are you should just be using the ParserOutput from
+	 * WikitextContent::getParserOutput instead of calling this for redirects.
+	 *
```

Comment:
The usage across the codebase matches the purpose of the method:
```
 130 ~/wikia/app $ grep -r viewRedirect *
extensions/wikia/EditPageLayout/EditPageLayoutAjax.class.php:                                                                           $html = $article->viewRedirect( array( $redirectTitle ) );
includes/Article.php:                                                   $wgOut->addHTML( $this->viewRedirect( $rt ) );
includes/Article.php:   public function viewRedirect( $target, $appendSubtitle = true, $forceKnown = false ) {
includes/ImagePage.php:                         $wgOut->addHTML( $this->viewRedirect( Title::makeTitle( NS_FILE, $this->mPage->getFile()->getName() ),
includes/EditPage.php:                          $previewHTML = $this->mArticle->viewRedirect( $rt, false );
```
:white_check_mark: